pkg/cmd/historical_logs.go

```diff
@@ -136,7 +136,7 @@ func (o *LogParameters) Execute(streams genericclioptions.IOStreams, args []stri
 
 }
 
-func fetchLogs(baseUrl string, logParameters *LogParameters, podname string, podLogsCh chan<- []string) {
+func FetchLogs(baseUrl string, logParameters *LogParameters, podname string, podLogsCh chan<- []string) {
```

Comment:
Should this method be public?

Reply:
Yes, it should be public for e2e tests. But for now I have removed that change and added it to e2e tests PR.